plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/catalog/glue/TestGlueCleanup.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.iceberg.catalog.glue;
+
+import com.amazonaws.services.glue.AWSGlueAsync;
+import com.amazonaws.services.glue.AWSGlueAsyncClientBuilder;
+import com.amazonaws.services.glue.model.Database;
+import com.amazonaws.services.glue.model.DatabaseInput;
+import com.amazonaws.services.glue.model.DeleteDatabaseRequest;
+import com.amazonaws.services.glue.model.GetDatabaseRequest;
+import com.amazonaws.services.glue.model.GetDatabasesRequest;
+import com.amazonaws.services.glue.model.GetDatabasesResult;
+import com.amazonaws.services.glue.model.UpdateDatabaseRequest;
+import com.google.common.collect.ImmutableMap;
+import io.airlift.log.Logger;
+import io.trino.plugin.hive.metastore.glue.GlueMetastoreApiStats;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+import java.util.Collection;
+import java.util.Date;
+
+import static io.trino.plugin.hive.metastore.glue.AwsSdkUtil.getPaginatedResults;
+
+/**
+ * Test class with methods to clean up existing Glue tables and schemas from test which were not finished properly.
+ */
+public class TestGlueCleanup
+{
+    // All tests creating Glue schemas should add this key/value pair to the schema properties.
+    public static final String GLUE_SCHEMA_PROPERTY_KEY = "ci_iceberg_integration_test";
```

Comment:
I find the property-based approach clever and neat.
I am concerned however that some new tests will forget to set it. Also, it's less obvious than a name. Simple persons like me will notice a name, but won't notice the propery.

I would rather have it based on schema name only.
It is not a hard ask to require each test schema name to begin with `test_`.

Reply:
I thought about that, but was worried about deleting schemas that might be used for other things. I could see there being `test_*` schemas that are supposed to be long-lived, but I don't know what else might be in the CI's AWS account.

Reply:
CI must run on an isolated account. 
tests may erroneously delete something, so you won't let tests run where you keep anything precious.

i agree `ci_` or `ci_test_` would be a better prefix, but we have already existing cases of using `test_` and am not worried about enforcing this in a generic fashion.

Reply:
> CI must run on an isolated account.
> tests may erroneously delete something, so you won't let tests run where you keep anything precious.

I guess I agree that you shouldn't run tests anywhere with precious databases, but when writing the tests I ran them against my personal AWS account, so I guess I wrote this with the same level of care as I would treat that.

Reply:
Please use your company's testing account when working locally. this will allow the test code to be simpler, and your personal AWS account to be safe. Still, removing `test_` schemas isn't "very destructive" even in a personal account.